src/html/tree.ml

```diff
@@ -337,11 +340,16 @@ let page_creator ?kind ?(theme_uri = Relative "./") ~path header_docs content =
           ] @
             (* Create breadcrumbs *)
             let space = Html.txt " " in
-            let init =
-              if !Relative_link.semantic_uris then "" else "index.html"
+            let breadcrumb_spec = match kind with
+            | Some `Page -> (fun n x -> n, dot, x)
```

Comment:
I would have expected the second element (here `dot`) to depend on `n`. Is this relying on an assumption about the nesting level of pages?

Reply:
Yes. Pages are flat at the moment and can only occur at the root of the package.